packages/apps/helpdesk/src/components/molecules/TicketCard/TicketCard.tsx

```diff
@@ -0,0 +1,172 @@
+  Icon,
+  LabelText,
+  Text,
+  useThemeColor,
+} from '@axelor/aos-mobile-ui';
+import {useTranslator, formatDateTime} from '@axelor/aos-mobile-core';
+import Ticket from '../../../types/ticket';
+
+interface TicketCardProps {
+  style?: any;
+  ticketSeq: string;
+  subject: string;
+  progressSelect: string;
+  duration: number;
+  deadlineDateT: string;
+  responsibleUser: string;
+  assignedToUser: string;
+  prioritySelect: number;
+  statusSelect: number;
+  allTicketType?: any;
+  ticketType?: any;
+  onPress: () => void;
+}
+const TicketCard = ({
+  style,
+  ticketSeq,
+  subject,
+  progressSelect,
+  duration,
+  deadlineDateT,
+  responsibleUser,
+  assignedToUser,
+  prioritySelect,
+  statusSelect,
+  allTicketType,
+  ticketType,
+  onPress,
+}: TicketCardProps) => {
+  const Colors = useThemeColor();
+  const I18n = useTranslator();
+
+  const colorType = useMemo(() => {
+    const colorIndex = allTicketType?.findIndex(
+      status => status.id === ticketType?.id,
+    );
+    return getStyles(Ticket.getTypeColor(colorIndex, Colors))?.colorBadge;
+  }, [Colors, allTicketType, ticketType?.id]);
+
+  const colorStatus = useMemo(() => {
+    return getStyles(Ticket.getStatusColor(statusSelect, Colors))?.colorBadge;
+  }, [Colors, statusSelect]);
+
+  const borderStyle = useMemo(() => {
+    return getStyles(Ticket.getPriorityColor(prioritySelect, Colors))?.border;
+  }, [prioritySelect, Colors]);
+
+  return (
+    <TouchableOpacity onPress={onPress} activeOpacity={0.9}>
+      <Card style={[styles.container, borderStyle, style]}>
+        <View style={styles.leftContainer}>
+          <Text style={styles.txtImportant}>
+            {I18n.t('Helpdesk_Progress')}n°{ticketSeq}
+          </Text>
+          {subject && <Text>{subject}</Text>}
+          {progressSelect && (
+            <Text>{`${I18n.t('Helpdesk_Progress')}: ${progressSelect} %`}</Text>
+          )}
+          {duration !== null && (
+            <Text>{`${I18n.t('Helpdesk_Duration')}: ${duration}`}</Text>
+          )}
+          {deadlineDateT && (
+            <LabelText
+              iconName="calendar-times"
+              title={`${I18n.t('Helpdesk_Deadline')} :`}
+              value={formatDateTime(
+                deadlineDateT,
+                I18n.t('Base_DateTimeFormat'),
+              )}
+            />
+          )}
+          {responsibleUser && (
+            <Text>{`${I18n.t(
+              'Helpdesk_User_In_Charge',
+            )}: ${responsibleUser}`}</Text>
+          )}
+          {assignedToUser && (
+            <Text>{`${I18n.t(
+              'Helpdesk_Assigned_To',
+            )}: ${assignedToUser}`}</Text>
+          )}
+        </View>
+        <View style={styles.rightContainer}>
+          <View style={styles.badgeContainer}>
+            <Badge
+              title={Ticket.getStatus(statusSelect, I18n)}
+              style={colorStatus}
+            />
+            <Badge title={ticketType?.name} style={colorType} />
```

Comment:
Badge component has color props, just give him the color and it makes the style itself